packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -114,9 +119,11 @@ import {
     chunksBlobName,
     electedSummarizerBlobName,
     gcFeature,
```

Comment:
Can we rename this to getGCVersion (or something similar)? The result of this is called GC version in this file. It's confusing to see two different things with the same name.

Reply:
Yeah, I was actually debating renaming it too.

Reply:
Changed it.